src/server/index.js

```diff
@@ -2,7 +2,7 @@
 const express = require('express');
 const cors = require('cors');
 const dotenv = require('dotenv');
-
+const {Router }=  require("express");
```

Comment:
Nit : Use lint before PR. Import statement should be clubbed together. Move it below db import